docs/02-for-app-authors/05-submission.md

```diff
@@ -1,8 +1,31 @@
 # Submission
 
-## How to submit an app
+App submissions are extremely welcome and the process is straightforward. Once you complete the steps below you can submit the app for inclusion.
+
+## Before submission
+
+Please follow the [requirements](/docs/for-app-authors/requirements) to ensure that the submission is technically and legally compatible with the Flathub service.
 
-App submissions are extremely welcome and the process is straightforward. Before submitting an app for inclusion on Flathub, please follow the [requirements](/docs/for-app-authors/requirements) to ensure that it is technically and legally compatible with the Flathub service. Once this has been done, you can submit the app for inclusion.
+It's best practice to make sure your submission correctly builds and runs locally with flatpak. This will help you to fix any potential issues and expedite the review process. Flathub recommends using the [flatpak package](https://github.com/flathub/org.flatpak.Builder) of flatpak-builder to build the application.
+```bash
+flatpak install -y flathub org.flatpak.Builder
+```
+1. To build and install the app:
+   ```bash
+   flatpak run org.flatpak.Builder --force-clean --sandbox --user --install --install-deps-from=flathub --install-deps-from=flathub-beta --mirror-screenshots-url=https://dl.flathub.org/repo/screenshots --repo=repo builddir <manifest>
```

Comment:
Wondering if `--install-deps-from=flathub-beta` is really necessary.

Also wondering, if we should add `--ccache`, but that's only useful for some apps I guess.

Reply:
```suggestion
   flatpak run org.flatpak.Builder --force-clean --sandbox --user --install --install-deps-from=flathub --ccache --mirror-screenshots-url=https://dl.flathub.org/repo/screenshots --repo=repo builddir <manifest>
```

Reply:
I tried to keep it mostly similar to what is run in build bot.

`--install-deps-from=flathub-beta` is for apps that might use runtimes from beta, maybe it can be referenced somewhere else.

Reply:
I think it's meant for the buildbot to not have different command line arguments.